lib/console_web/controllers/gateway_controller.ex

```diff
@@ -61,6 +61,34 @@ defmodule ConsoleWeb.GatewayController do
     end
   end
 
+  def register(conn, %{"OUI" => oui, "nonce" => nonce, "gateway" => gateway}) do
```

Comment:
you probably want to pattern match into the gateway as well, so that it's super clear what's coming through there

Reply:
sounds good